.github/workflows/deployment_prs.sh

```diff
@@ -19,30 +19,30 @@ cd -
 make travis_docker_push
```

Comment:
@syed-awais-ali One last thing, could you update this to point at `docker_push`?